### PR TITLE
{bio}[foss/2023a] BindCraft v1.1.0 w/ CUDA 12.1.1

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/BindCraft/BindCraft-1.1.0-foss-2023a-CUDA-12.1.1.eb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    ('matplotlib', '3.7.2'),
+    ('PyRosetta', '4.release-387'),
+    ('jax', '0.4.25', versionsuffix),
+    ('dm-haiku', '0.0.12', versionsuffix),
+    ('dm-tree', '0.1.8'),
+    ('ml-collections', '0.1.1'),
+    ('Optax', '0.2.2', versionsuffix),
+    ('py3Dmol', '2.1.0'),
+    ('JupyterLab', '4.0.5'),
+    ('hatchling', '1.18.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/b/BindCraft/BindCraft-1.1.0-foss-2023a-CUDA-12.1.1.eb:35]: Make this a build-only dependency

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3-a100

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**